collections/_posts/2024-04-18-DehazingDiffusion.md

```diff
@@ -0,0 +1,380 @@
+---
+layout: review
+title: "Dehazing Ultrasound using Diffusion Models"
+tags: diffusion-models, ultrasound-imaging, denoising
+author: "Julia Puig"
+cite:
+    authors: "Tristan S.W. Stevens, Faik C. Meral, Jason Yu, Iason Z. Apostolakis Jean-Luc Robert and Ruud J.G. van Sloun"
+    title:   "Dehazing Ultrasound using Diffusion Models"
+    venue:   "IEEE Transactions on Medical Imaging (2023)"
+pdf: "http://arxiv.org/abs/2307.11204"
+---
+
+<br/>
+
+# Ultrasound image quality
+During ultrasound acquisitions, phenomena such as aberration and reverberation produce unwanted echoes that degrade the image quality. In particular, **haze** is an artifact that occurs due to multipath reflections and produces a white haze on the echo image.
+
+{:refdef: style="text-align: center;"}
+![](/collections/images/DehazingDiffusion/echo_images.jpg){: width="500" }
+{:refdef}
+
+This phenomenon can highly **difficult diagnosis** both qualitative and quantitative.
+
+Existing dehazing techniques:
+- **Harmonic imaging** receives echoes at frequencies that are multiples of the original frequency. It produces higher quality images, as multipath scatterers have less energy and therefore generate fewer harmonics. However, it results in reduced penetration depth and image frame rate due to the additional pulse inversion transmits.
```

Comment:
Is _transmits_ (at the end) the right word? Could it be something like _transmission_?

Reply:
yes indeed it should be that way